data/json/mapgen/nether_monster_corpse/monster_bowels.json

```diff
@@ -0,0 +1,313 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "corpse_bowels_mid",
+    "object": {
+      "fill_ter": "t_dirt_underground",
+      "rows": [
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "terrain": { "#": "t_soil" }
```

Comment:
I won't populate these 2 OMTs in this PR, as this will be the focus of the next PR. Ideally they should both have specific structures and variations within them, with different organs/scavenger nests.